src/services/EdFi.DataManagementService.Api/Program.cs

```diff
@@ -6,6 +6,10 @@
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
+app.UsePathBase("/api");
+app.UseRouting();
+
 app.MapGet("/", () => "Data Management Service");
+app.MapGet("/ping", () => Results.Ok(new { StatusCode = 200, CurrentDateTime = DateTime.Now.ToString() }));
```

Comment:
Suggestion, it's not necessary to send the status code in the response, it's taken care of with `Results.Ok`.

Reply:
> Suggestion, it's not necessary to send the status code in the response, it's taken care of with `Results.Ok`.

True, just created custom response with status code.